pkg/crc/preflight/preflight_windows.go

```diff
@@ -140,38 +174,37 @@ var vsockChecks = []Check{
 	},
 }
 
-var adminHelperChecks = []Check{
-	{
-		configKeySuffix:  "check-crc-users-group-exists",
-		checkDescription: "Checking if crc-users group exists",
-		check: func() error {
-			_, _, err := powershell.Execute("Get-LocalGroup -Name crc-users")
-			return err
-		},
-		fixDescription: "Creating crc-users group",
-		fix: func() error {
-			_, _, err := powershell.ExecuteAsAdmin("create crc-users group", "New-LocalGroup -Name crc-users")
-			return err
-		},
-		labels: labels{Os: Windows},
+var errReboot = errors.New("Please reboot your system and run 'crc setup' to complete the setup process")
+
+var adminHelperGroup = Check{
```

Comment:
Does this bring any benefits to keep using a `Check` for this? I don't think `configKeySuffix` will still be used?
Looks like you could call `checkIfUserPartOfHyperVAdmins` directly, and have similar helpers for the `crc-users` group?

Concerns I have are around the `skip-` config options. I think after this `check-user-in-hyperv-group` will no longer be a valid config option? And doing 2 very different things in a single preflight check is problematic from the perspective of  these skip options, if only one of these groups is undesirable, as a user I can no longer skip the corresponding test.

Reply:
> This line is also weird: https://github.com/code-ready/crc/blob/master/pkg/crc/preflight/preflight_checks_windows.go#L99

It's not really possible to abort preflight checks during setup, they will all run even if their `fix` method returns an error, so this `abort()` was added to stop as soon as we now a reboot will be needed.
One PR I've had around for far too long is one adding some kind of state to all preflights ( https://github.com/cfergeau/crc/commit/04ec76c308486cb4176e8cf199520380dd37d3b1 adds an 'options' parameter, but this could be a receiver instead). If we had some preflight state, we could do `SetRebootNeeded(...)` in the relevant preflight, and then have one test which aborts/fails when `RebootNeeded` is set. But we are not there yet :-/

Reply:
No real good solution. Can we keep this as it is for the current release ?

Reply:
This is breaking backwards-compatibility (removal of a config option), and this ties 2 distinct things in a single preflight, which makes it harder to skip just the hyperv group if this was problematic before. So this is not as simple as "this is an improvement over the previous situation, we can refine it later", as it is now, it is both an improvement and a regression.

Reply:
OK so to get that in the release, I won't have time to get something proper running. Let's remove the last commit.
The user will have to reboot twice his computer for installing crc.

Reply:
Maybe for now the new user can be added as part of `check-user-in-hyperv-group` so that we at least keep the config key naem, and we improve on this next release?

Reply:
So this is equivalent to https://github.com/code-ready/crc/commit/3dd44303afaa85fb183200a3610da7dcf4635608 + change the config key  "check-user-in-groups" to "check-user-in-hyperv-group".
Should I push this?

Reply:
yeah, this might be the 'less bad' option, but maybe you feel strongly otherwise?

Reply:
Ok done. I dont really have strong feelings about this. I just want to get the release done in time :)

Reply:
I think this can be squashed in the previous commit?

Reply:
The commit itself explains why this is ugly. It also avoids a too large commit.

Reply:
Also later we'll know why these two checks were combined into one, (to avoid rebooting twice)